internal/client/vpc.go

```diff
@@ -25,11 +25,11 @@ type VPC struct {
 }
 
 type PeeringConnection struct {
-	ID           string     `json:"id"`
-	VPCID        string     `json:"vpcId"`
-	Status       string     `json:"status"`
-	ErrorMessage string     `json:"errorMessage"`
-	PeerVPCs     []*PeerVPC `json:"peerVPC"`
+	ID           string   `json:"id"`
+	VPCID        string   `json:"vpcId"`
+	Status       string   `json:"status"`
+	ErrorMessage string   `json:"errorMessage"`
+	PeerVPCs     *PeerVPC `json:"peerVPC"`
```

Comment:
```suggestion
	PeerVPC     *PeerVPC `json:"peerVPC"`
```
does this change look right?